pkg/operator/ceph/object/bucket/s3-handlers.go

```diff
@@ -176,6 +176,7 @@ const (
 )
 
 // AllowedActions is a lenient default list of actions
+// Removing GetBucketPolicy from the list, refer the commit 43112281
```

Comment:
Why do we need this comment? Also the commit ID seems incorrect.

Reply:
Done